### DISCUSSION TITLE
브라우저에 주식 포트폴리오가 렌더링 되기 까지

### DISCUSSION THREAD

**Discussion by TakYeji** [100-hours-a-week/discussions#23] [Deepdive]

**Comment by ej31** [comment]
# 답변
- 해소가 필요한 모호함이 있습니다.
  - 딥다이브를 하려면 특정 주제에 대한 깊은 분석이 필요합니다.
  - `모의 주식 포트폴리오 시각화 최적화` 라는 주제는 UX 관점에서 최적화를 한다는 것이고 성능과는 별 연관이 없어보이는 상태입니다.
  - 만약 `모의 주식 포트폴리오 시각화 최적화` 주제를 통해 딥다이브를 진행한다면 사용자의 경험이 얼만큼, 몇 퍼센트나 개선되었는지 정량적인 수치화가 가능해야 할 것 같지만 현실적으로 사용자 그룹 만들어서 조사하는 등의 행동은 현재 하기 어려운 시점입니다.

## 추천 사항
- 개인적인 추천사항은 다음과 같습니다.

**1. 브라우저에 주식 포트폴리오가 렌더링 되기 까지의 과정을 딥다이브 해보시는 걸 추천드립니다.**
2. 주제에서 최적화라는 맥락은 없애고 렌더링 자체가 되는 과정을 딥다이브 해보시게 된다면 그 자체만으로 쓸 내용이 많습니다.
3. 또한 렌더링 되는 과정에 대한 딥다이브가 생각보다 일찍 끝났다면 작성하던 주제에 이어서 렌더링 되는 시간을 줄인다던지 혹은 다른 라이브러리를 적용했을 때에는 렌더링 시 램 사용량 및 로드 타임 등 성능차이가 얼마나 나고 왜 그런 결과가 나오는지 분석해서 이어 나가면 좋을 것으로 보입니다.
